pkg/tuned/dialog.jsx

```diff
@@ -314,6 +313,7 @@ const TunedDialog = ({
                    </>
                }
         >
+            {error && <ModalError dialogError={typeof error == 'string' ? error : error.message} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-184951-351895da-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)